api/src/services/users.js

```diff
@@ -515,7 +515,7 @@ module.exports = {
         !_.isNull(data.contact) &&
         !_.some(props, key => (!_.isNull(data[key]) && !_.isUndefined(data[key])))
     ) {
-      return Promise.reject(error400('One of the following fields are required: ' + props.join(', ')));
+      return Promise.reject(error400({ message: 'One of the following fields are required: ' + props.join(', '), translationKey: 'fields.one.required '}));
```

Comment:
We could add a translation parameter here to list the missing required fields in the translation as well.